.editorconfig

```diff
@@ -48,15 +48,30 @@ indent_size = 2
 end_of_line = lf
 indent_size = 2
 
+# .github workflows
+[*.{yaml, yml}]
+end_of_line = lf
+indent_size = 2
+
 # Dotnet code style settings:
 [*.{cs,vb}]
+
+// Missing XML comment for publicly visible type or member
+dotnet_diagnostic.CS1591.severity = none
+
 # Member can be made 'readonly'
 csharp_style_prefer_readonly_struct_member = true
 dotnet_diagnostic.IDE0251.severity = warning
 dotnet_diagnostic.IDE0044.severity = warning
 
 dotnet_diagnostic.CS1591.severity = silent
 
+# Avoid problematic synchronous waits
+dotnet_diagnostic.VSTHRD002.severity = none
```

Comment:
Change the level to warning?

Reply:
This can break code; if you do `dotnet format`.